### PR TITLE
docs(connector): apple readme (zh-CN)

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

Apple 连接器

## Testing

N/A

### CODE REVIEW

File: packages/connector-apple/README.md [hunk 1]
```diff
@@ -76,3 +84,69 @@ You need to use the identifier that fills in the [Create an identifier](#create-
...
+
+![Enable Sign in with Apple](/packages/connector-apple/docs/enable-sign-in-with-apple-in-xcode.png)
+
+参见 [Apple 官方文档](https://developer.apple.com/documentation/sign_in_with_apple/configuring_your_environment_for_sign_in_with_apple) 里的「Enable an App ID」章节以了解更多。
+
+### 创建一个 identifier
+
+1. 访问 [Certificates, Identifiers & Profiles](https://developer.apple.com/account/resources/identifiers/list/serviceId)，并点按在「Identifier」旁边的「+」按钮。
+2. 在「Register a new identifier」页面，选择「Services IDs」并点按「Continue」。
+3. 填写「Description」与「Identifier」（例如 `Logto Test` 和 `io.logto.test`），并点按「继续」。
```
Comment (IceHe) [inline packages/connector-apple/README.md:117]: `继续` should be `Continue` as below?
```suggestion
3. 填写「Description」与「Identifier」（例如 `Logto Test` 和 `io.logto.test`），并点按「Continue」。
```

File: packages/connector-wechat/README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The official Logto connector for WeChat social sign-in in web apps.
 
 ## Get started
 
-If you don't know the concept of the connector or haven't added this connector to your Sign-in experience yet, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
+If you don't know the concept of the connector or don't how to added this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```
Comment (IceHe) [inline packages/connector-wechat/README.md:32]:
```suggestion
If you don't know the concept of the connector or don't how to add this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```

File: packages/connector-apple/README.md [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+    - [为你的应用启用「通过 Apple 登录」](#为你的应用启用通过-apple-登录)
+    - [创建一个 identifier](#创建一个-identifier)
+    - [为你的 identifier 启用「通过 Apple 登录」](#为你的-identifier-启用通过-apple-登录)
+  - [编写连接器的 JSON](#编写连接器的-json)
+  - [测试 Apple 连接器](#测试-apple-连接器)
 
 ## Get started
 
-If you don't know the concept of the connector or haven't added this connector to your Sign-in experience yet, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
+If you don't know the concept of the connector or don't how to added this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```
Comment (darcyYe) [inline packages/connector-apple/README.md:26]: missing `know` after `don't`?
```suggestion
If you don't know the concept of the connector or don't know how to add this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```

File: packages/connector-wechat-native/README.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ The official Logto connector for WeChat social sign-in in native apps (iOS and A
 
 ## Get started
 
-If you don't know the concept of the connector or haven't added this connector to your Sign-in experience yet, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
+If you don't know the concept of the connector or don't how to added this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```
Comment (darcyYe) [inline packages/connector-wechat-native/README.md:39]:
```suggestion
If you don't know the concept of the connector or don't know how to add this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```

File: packages/connector-wechat/README.md (hunk 2 above)
Comment (darcyYe) [inline packages/connector-wechat/README.md:32]:
```suggestion
If you don't know the concept of the connector or don't know how to add this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```

**Review by darcyYe — APPROVED** [review]
Overall LGTM

**Pull request merged**